handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslContext.java

```diff
@@ -487,12 +491,36 @@ public final long sslCtxPointer() {
         return ctx;
     }
 
+    /**
+     * Enables OCSP stapling or throws an {@link Exception} if OCSP is not supported.
+     */
+    @UnstableApi
+    public final void enableOcsp() {
```

Comment:
We can move it into SslContextBuilder and it's only allowed to transition from disabled to enabled.